jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -17,15 +17,31 @@ pipeline {
 
         stage('Clone Operator Repo') {
             steps {
-                //ToDo Clone the integeratly-operator repo
-                echo "Clone Operator Repo"
+                cleanWs()
+                git 'https://github.com/integr8ly/integreatly-operator'
```

Comment:
This is nice. Is there more ways to use this other than cloning? I didn't like using so much `sh` everywhere but didn't know that the clone could be done like this.

Reply:
I'm not sure really, here's the docs https://jenkins.io/doc/pipeline/steps/git/